cmd/gophermart/main.go

```diff
@@ -1,3 +1,121 @@
 package main
 
-func main() {}
+import (
+	"github.com/go-chi/chi/v5"
+	"gophermart/internal/config"
+	"gophermart/internal/handlers"
+	"gophermart/internal/middleware"
+	"gophermart/internal/repository"
+	"gophermart/internal/service"
+	"gophermart/storage"
+	"gorm.io/driver/postgres"
+	"gorm.io/gorm"
+	"log"
+	"net/http"
+)
+
+func main() {
+	PgsStorage := &storage.PgStorage{}
```

Comment:
Локальные переменные нет смысла именовать с заглавной буквы, напротив стоит именовать из с буквы в нижнем регистре.
```suggestion
	pgsStorage := &storage.PgStorage{}
```